pkg/install/install.go

```diff
@@ -28,24 +28,20 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
-func Run(automatic bool, configFile string, powerOff bool, silent bool) error {
+func Run(automatic bool, configFile string, powerOff bool, reboot bool, silent bool) error {
 	cfg, err := config.ReadConfig(context.Background(), configFile, automatic)
 	if err != nil {
 		return err
 	}
 
-	if powerOff {
-		cfg.RancherOS.Install.PowerOff = true
-	}
+	cfg.RancherOS.Install.PowerOff = powerOff
+	cfg.RancherOS.Install.Reboot = reboot
+	cfg.RancherOS.Install.Automatic = silent
 
 	if automatic && !cfg.RancherOS.Install.Automatic {
 		return nil
 	}
 
-	if silent {
```

Comment:
I think this construct was there to make sure that the CLI overrides the config only to enable certain aspects, but the config can still be configured independently. That is, if the user runs the installer without silent, or poweroff, the config section takes precedence. In this way it would completely override what's specified in the config

Reply:
goddammit, everything about this is wroooong